Source/UI/NewUI/BillingForm/ViewModel/DefaultBillingFormViewModel.swift

```diff
@@ -59,58 +59,90 @@ final class DefaultBillingFormViewModel: BillingFormViewModel {
         editDelegate = view
         return view
     }
-
-    func getCell(tableView: UITableView, indexPath: IndexPath,  delegate: CellTextFieldDelegate?) -> UITableViewCell {
-        guard style.cells.count > indexPath.row, var viewStyle = style.cells[indexPath.row].style
-        else { return UITableViewCell() }
-
-        updateStyle(with: &viewStyle, cellStyle: style.cells[indexPath.row], tag: indexPath.row)
-
-        return getCell(tableView: tableView,
-                       viewStyle: &viewStyle,
-                       cellStyle: style.cells[indexPath.row],
-                       indexPath: indexPath,
-                       delegate: delegate)
+    
+    func getCell(tableView: UITableView, indexPath: IndexPath, sender: UIViewController?) -> UITableViewCell {
+        guard style.cells.count > indexPath.row else { return UITableViewCell() }
+        
+        if style.cells[indexPath.row].index == BillingFormCell.country(nil).index {
+            return getCountryCell(tableView: tableView, indexPath: indexPath, sender: sender)
+        }
+        return getTextFieldCell(tableView: tableView, indexPath: indexPath, sender: sender)
     }
-
+    
     // MARK: - Private methods
-
-    private func updateStyle(with viewStyle: inout CellTextFieldStyle, cellStyle: BillingFormCell, tag: Int) {
-        if viewStyle.isOptional {
-            textValueOfCellType[cellStyle.index, default: ""] += ""
-            errorFlagOfCellType[cellStyle.index] = false
+    
+    /// Text field cell
```

Comment:
Do we need this comment the function tells clearly its getT ext Field Cell, or is there some other purpose of this comment, Same applies to other 2 below /// comments